pkg/cli/build.go

```diff
@@ -0,0 +1,416 @@
+			return options, nil, err
+		}
+	}
+	var timestamp *time.Time
+	if c.Flag("timestamp").Changed {
+		t := time.Unix(iopts.Timestamp, 0).UTC()
+		timestamp = &t
+	}
+	if c.Flag("output").Changed {
+		buildOption, err := parse.GetBuildOutput(iopts.BuildOutput)
+		if err != nil {
+			return options, nil, err
+		}
+		if buildOption.IsStdout {
+			iopts.Quiet = true
+		}
+	}
+	options = define.BuildOptions{
+		AddCapabilities:         iopts.CapAdd,
+		AdditionalTags:          tags,
+		AllPlatforms:            iopts.AllPlatforms,
+		Annotations:             iopts.Annotation,
+		Architecture:            systemContext.ArchitectureChoice,
+		Args:                    args,
+		AdditionalBuildContexts: additionalBuildContext,
+		BlobDirectory:           iopts.BlobCache,
+		CNIConfigDir:            iopts.CNIConfigDir,
+		CNIPluginPath:           iopts.CNIPlugInPath,
+		CommonBuildOpts:         commonOpts,
+		Compression:             compression,
+		ConfigureNetwork:        networkPolicy,
+		ContextDirectory:        contextDir,
+		CPPFlags:                iopts.CPPFlags,
+		Devices:                 iopts.Devices,
+		DropCapabilities:        iopts.CapDrop,
+		Err:                     stderr,
+		ForceRmIntermediateCtrs: iopts.ForceRm,
+		From:                    iopts.From,
+		IDMappingOptions:        idmappingOptions,
+		IIDFile:                 iopts.Iidfile,
+		In:                      stdin,
+		Isolation:               isolation,
+		IgnoreFile:              iopts.IgnoreFile,
+		Labels:                  iopts.Label,
+		Layers:                  layers,
+		LogRusage:               iopts.LogRusage,
+		Manifest:                iopts.Manifest,
+		MaxPullPushRetries:      MaxPullPushRetries,
+		NamespaceOptions:        namespaceOptions,
+		NoCache:                 iopts.NoCache,
+		OS:                      systemContext.OSChoice,
+		Out:                     stdout,
+		Output:                  output,
+		BuildOutput:             iopts.BuildOutput,
+		OutputFormat:            format,
+		PullPolicy:              pullPolicy,
+		PullPushRetryDelay:      PullPushRetryDelay,
+		Quiet:                   iopts.Quiet,
+		RemoveIntermediateCtrs:  iopts.Rm,
+		ReportWriter:            reporter,
+		Runtime:                 iopts.Runtime,
+		RuntimeArgs:             runtimeFlags,
+		RusageLogFile:           iopts.RusageLogFile,
+		SignBy:                  iopts.SignBy,
+		SignaturePolicyPath:     iopts.SignaturePolicy,
+		Squash:                  iopts.Squash,
+		SystemContext:           systemContext,
+		Target:                  iopts.Target,
+		TransientMounts:         iopts.Volumes,
+		OciDecryptConfig:        decryptConfig,
+		Jobs:                    &iopts.Jobs,
+		Excludes:                excludes,
+		Timestamp:               timestamp,
+		Platforms:               platforms,
+		UnsetEnvs:               iopts.UnsetEnvs,
+		Envs:                    iopts.Envs,
+		OSFeatures:              iopts.OSFeatures,
+		OSVersion:               iopts.OSVersion,
```

Comment:
If we're going to make this massive change that can't be reviewed easily anyway, I'd like to alphabetize these fields in here.  @nalind may have other opinions.

Reply:
I think we can alphabetize since we break git split now.